README.md

```diff
@@ -27,3 +27,31 @@ e.g.
 ![Docker Hub](img/dockerhub.png)
 
 ### 4. Push image to Docker Hub
+
+## Running as a Docker Container
+
+You can also run this app as a Docker container instead of deploying it to Heroku.
+
+### 1. Generate Docker Image
+
+```
+docker build -t dockerhub-slack-webhook .
```

Comment:
This step is needless 

c.f. https://github.com/sue445/dockerhub-slack-webhook/pull/223#discussion_r349905796